README.md

```diff
@@ -9,22 +9,44 @@ npm install -g tattoo
 
 ### Usage
 
-***Test all repos***
-```
-mkdir tattoo_work
-cd tattoo_work
-tattoo
-```
+When you run tattoo, it will create and/or reuse a directory called
+`./tattoo_workspace`, relative to the current working dir.  This folder is
+used to clone test target repositories and their dependencies.  You can change
+the directory used with the `--workspace-dir` or `-w` option.
```

Comment:
No.  However, I added Issue #23 to support local filesystem references so doing what you say would look like `tattoo MyOrg/*-* -r .` would run tests for all of MyOrg's custom-element repos with the local folder as one of the dependency repos.